yarn.lock

```diff
@@ -0,0 +1,3744 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why do we need both a `yarn.lock` and a `package-lock.json`?  Generally speaking you only use one or the other...

Reply:
I'll blow this away. I believe its from `yarn link`.